README.md

```diff
@@ -36,7 +36,7 @@ Prerequisites
 
 For Ginkgo core library:
 
-*   _cmake 3.9+_
+*   _cmake 3.13+_
 *   C++14 compliant compiler, one of:
     *   _gcc 5.3+, 6.3+, 7.3+, all versions after 8.1+_
     *   _clang 3.9+_
```

Comment:
Maybe mention explicitly which compilers we are running in CI, together with a sentence like `Compilation might succeed on older compilers, but we do not monitor their status actively`. The GCC list is very verbose compared to the rest, maybe we can shorten it to something like 5.3+?

Reply:
The problem is the GCC release system, which isn't linear. e.g., 5.5 was released after 7.2 :)
https://gcc.gnu.org/releases.html

We tested and fixed so that the ones listed here work, but we have no idea for later ones. It's true though since in practice we only test 5.5 we might have lost 5.4 or 5.3 compatibilities somewhere down the line... In short, it's a mess and I'm also not sure if listing what we test would make this any clearer (particularly since it might change at every container update).

Reply:
I think I will go with 5.5+ which we are more sure about and are still testing.